### PR TITLE
Feat/onepage

### PR DESCRIPTION
**Pull request by ElTimuro**
moving toward the final ux

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

File: apps/web/src/app/(dashboard)/documents/page.tsx [hunk 1]
```diff
@@ -110,21 +98,30 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
...
-            Add Document
-          </Button>
         </div>
       </div>
 
       <div className="mt-8">
-        {/* If we're viewing all documents for all time and there's nuffin we should should an add document component instead */}
+        {/* If we're viewing all documents for all time and there's nuffin we should should show an add document component instead */}
         {isNoResults ? (
           <DocumentDropzone className="min-h-[60vh] md:min-h-[40vh]" />
```
Comment (Mythie) [inline apps/web/src/app/(dashboard)/documents/page.tsx:126]: Wouldn't make sense to show this twice now would it 🤔

Comment (ElTimuro) [reply apps/web/src/app/(dashboard)/documents/page.tsx:126]: Interesting point. Considering the massive CTA above, another button (even if small) seems redundant.

Comment (ElTimuro) [reply apps/web/src/app/(dashboard)/documents/page.tsx]: Removed it

**Pull request merged**